quotes.html

```diff
@@ -0,0 +1,88 @@
+<!DOCTYPE html>
+	<html>
+	<head>
+		<link rel="stylesheet" type="text/css" href="css/styles.css">
+		<title>KANYE RESUME</title>
+	</head>
+
+	<body>
+		<nav></nav>
+		<nav></nav>
+			
+			<div class="header">
+				<h1>KANYE WEST</h1>
+
+				<ul>
+					<li style="color: black">QUOTES</li>
+					<li style="color: black"><b>||</b></li>
+					<li>
+						<a href="index.html">ME, MYSELF & I
+					</li> 
+				</ul>
+			</div>
+		
+		<nav></nav>
+
+			<div>
```

Comment:
@humphriescarly These `<div>`'s with `<h5>`'s in them are not needed. Remember, heading tags are _block_ elements just like `<div>`'s are, so you don't have to wrap them in anything. They will create their own line by themselves.